core/src/serialize.rs

```diff
@@ -47,6 +46,25 @@ impl fmt::Display for ExportFormat {
     }
 }
 
+/// Available common export formats.
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Default, clap::ValueEnum)]
+pub enum ExportFormatCommon {
```

Comment:
`ExportFormatCommon` isn't very telling, IMHO (how it is different than `ExportFormat` or other formats described elsewhere in the codebase?). If there's no good general way to describe it, maybe something more descriptive such as `MetadataExportFormat` is less confusing?